core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -724,9 +730,9 @@ class TransactionsTest extends IntegrationTestHarness {
       producer.beginTransaction()
       producer.send(TestUtils.producerRecordWithExpectedTransactionStatus(topic1, null, "2", "2", willBeCommitted = false))
 
-      killBroker(partitionLeader) // kill the partition leader to prevent the batch from being submitted
```

Comment:
was there a reason we removed the comments here?

Reply:
ah yes I thought it was self explanatory and that I added them haha, but I can add em back